simulator/src/main/java/com/hazelcast/simulator/worker/MemberWorker.java

```diff
@@ -117,6 +119,29 @@ private HazelcastInstance createClientHazelcastInstance() throws Exception {
         return client;
     }
 
+    private void warmupPartitions(Logger logger, HazelcastInstance hz) {
```

Comment:
The method was `static` and in a util class, which was moved into the utils module. This method couldn't go, because it has a Hazelcast dependency. `MemberWorker` is also the only class which used this method, so I parked it here.